src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text, Icon, FixedIcon, Title, Button } from '@gnosis.pm/safe-react-components'
+
+const Wrapper = styled.div`
+  width: 100%;
+  margin-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+`
+
+const Content = styled.div`
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  > * {
+    margin-top: 10px;
+  }
+`
+
+const LinkWrapper = styled.div`
+  display: inline-flex;
+  margin-bottom: 10px;
+
+  > :first-of-type {
+    margin-right: 5px;
+  }
+`
+
+const StyledButton = styled(Button)`
+  span.MuiButton-label {
+    svg {
+      margin-right: 5px;
+    }
+  }
+`
+
+type Props = {
+  error: Error
+  componentStack: string
+  resetError: () => void
+}
+
+const GlobalErrorBoundaryFallback = ({ error, componentStack }: Props): React.ReactElement => {
+  const isProduction = process.env.NODE_ENV === 'production'
```

Comment:
I would move this to the constants file so in case we need to use it later we can access directly to `isProduction`

Reply:
good idea